packages/opentelemetry-node/lib/elastic-node-sdk.js

```diff
@@ -53,7 +55,7 @@ class ElasticNodeSDK extends NodeSDK {
         // Get logs exporter protocol based on environment.
         const logsExportProtocol =
             process.env.OTEL_EXPORTER_OTLP_LOGS_PROTOCOL ||
-            getEnvVar('OTEL_EXPORTER_OTLP_PROTOCOL');
+            getEnvString('OTEL_EXPORTER_OTLP_PROTOCOL');
```

Comment:
```
% node --import @elastic/opentelemetry-node simple-http-request.js
{"name":"elastic-otel-node","level":40,"msg":"Logs exporter protocol \"undefined\" unknown. Using default \"http/protobuf\" protocol","time":"2025-03-18T20:10:04.261Z"}
...
```

I think that log warn is because this doesn't have a default fallback now.